core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -46,10 +46,10 @@ angular.module('oppia').factory('SkillEditorStateService', [
       current: [],
       others: []
     };
-    var skillChangedSubject = new Subject();
+    var skillChangedSubject = new EventEmitter();
```

Comment:
Do you still want to call this "subject"?

Maybe we should call it skillChangedEventEmitter (or similar)?

Reply:
Changed it to skillChangedEvent

Reply:
But it's not an event. It's an event emitter.

Reply:
changed it to skillChangedEventEmitter